jobs/kpi-forecasting/kpi-forecasting/Utils/PosteriorSampling.py

```diff
@@ -83,10 +84,12 @@ def get_confidence_intervals(
         ).dayofyear
         != 1
     ):
-        uncertainty_samples_aggregated.at[0, 1:] = (
-            uncertainty_samples_aggregated.iloc[0, 1:]
-            + observed_aggregated.iloc[-1].value
-        )
+        # add observed samples from current time period to uncertainty samples for
+        # the remainder of the period.
+        uncertainty_samples_aggregated.iloc[0, 1:] += observed_aggregated["value"].iloc[
+            -1
+        ]
```

Comment:
This is the same intended logic as before, but the previous code doesn't work in new versions of pandas because `observed_aggregated.iloc[-1].value` doesn't return a single value, it returns an array of values. Using the `.` column access method was also confusing, because at first glance it looks like a typo of `.values` which casts a pandas column to a numpy array.